crates/gen/src/types/type_name.rs

```diff
@@ -354,12 +369,7 @@ impl TypeName {
     /// definition tokens and regular tokens. Definition tokens are those that require a
     /// colon spearater between the name and the generics (e.g., `Vector::<OtherType>`) vs.  
     /// `Vector<OtherType>`
-    fn generate_tokens<F>(
-        &self,
-        namespace: Option<&TokenStream>,
-        calling_namespace: &str,
-        format: F,
-    ) -> TokenStream
+    fn generate_tokens<F>(&self, namespace: Option<&TokenStream>, format: F) -> TokenStream
```

Comment:
Why two generate_tokens functions?

Reply:
No reason. Just ran out of steam refactoring this monster PR. Will clean this up soon.

Reply:
Done.